src/ypd/model/project.py

```diff
@@ -105,6 +105,15 @@ def edit(self, user, session=None, **kwargs):
                 setattr(self, name, value)
         session.add(self)
 
+    @SessionManager.with_session
+    def archive_project(self, user, session=None):
+        """"Archives the project"""
```

Comment:
This function doesn't archive the project. This functions toggles the archived flag. The name is confusing, and the docstring is inaccurate

Reply:
Changed function to be toggle_archived and comment to reflect. Made ability to restore project to Archived = False only for admins.